snippets/ztp_example-sno.yaml

```diff
@@ -72,8 +72,8 @@ spec:
           bmcCredentialsName:
             name: "example-node1-bmh-secret"
           bootMACAddress: "AA:BB:CC:DD:EE:11"
-          # Use UEFISecureBoot to enable secure boot
-          bootMode: "UEFI"
+          # Use UEFISecureBoot to enable secure boot, UEFI to disable.
```

Comment:
We need to confirm this part, iirc setting `UEFI` mode doesn't disable previously/already configured _SecureBoot_  so this should be done manually

Reply:
@imiller0 can you comment?

Reply:
Agreed. Poorly worded comment. I believe that UEFI simply leaves secureboot "as is", whereas "UEFISecureBoot" enables it. I will revert that comment to "# Use UEFISecureBoot to enable secure boot"